addons/udes_stock/models/procurement.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+ 
+from odoo import api, models, _
+from odoo.exceptions import ValidationError
+
+class ProcurementGroup(models.Model):
+    _name = 'procurement.group'
```

Comment:
Is `_name` really needed?

Reply:
This also applies to the other models :)

Reply:
I think it is in this case, as we _inherit from multiple models.